src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1003,6 +1011,10 @@ private void AddSpecialInterface(ref ListBuilder<RuntimeType> list, Filter filte
                     }
                 }
 
+                [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2065:UnrecognizedReflectionPattern",
```

Comment:
Can we put `[DynamicallyAccessed(Interfaces)]` on the method?

Reply:
It's very tricky. This all rolls up to `RuntimeTypeCache.m_runtimeType` which would have to be annotated, but that is called for example from `Type.GetProperties` which is annotated with `Properties`. The suppression would have to be somewhere. This all boils down to the fact that the `RuntimeTypeCache` caches everything in one class - we fill it piecemeal (if fields are needed we fill fields, later on we may add methods and so on), but statically it can contain all of the member types at once, and so it would need to be annotated as such.

The reason why this is the first time we hit this is that we didn't annotate the runtime calls in reflection as "dangerous - for example RuntimeTypeHandle.GetNextIntroducedMethod should be "dangerous" as in requiring all methods on the type. But annotating all of this is tricky and will eventually all clash inside the `RuntimeTypeCache` anyway.